app/controllers/administrative-units/administrative-unit/local-involvements/edit.js

```diff
@@ -66,13 +73,21 @@ export default class AdministrativeUnitsAdministrativeUnitLocalInvolvementsEditC
     this.showTotalFinancingPercentageError = false;
   }
 
+  hideMoreThanOneFinancialTypeError() {
+    this.showMoreThanOneFinancialTypeError = false;
+  }
+
   @action
   handleInvolvementTypeSelection(involvement, involvementType) {
     involvement.involvementType = involvementType;
 
     if (!involvementType || involvementType.id !== INVOLVEMENT_TYPE.FINANCIAL) {
       involvement.percentage = 0;
     }
+
+    if (this.isLessOrOneFinancialLocalInvolvement) {
```

Comment:
Shouldn't `isLessOrOneFinancialLocalInvolvement` is `isOneOrLessFinancialLocalInvolvement` ? Pure syntaxic question but "less or one" doesn't make sense no ? :sweat_smile:

Reply:
Indeed! I renamed it to `isOneOrLessFinancialLocalInvolvement`